transformer_engine/pytorch/csrc/extensions/attention.cu

```diff
@@ -83,9 +83,8 @@ std::vector<at::Tensor> fused_attn_fwd_qkvpacked(
     NVTE_QKV_Layout qkv_layout, NVTE_Bias_Type bias_type, NVTE_Mask_Type attn_mask_type,
     const std::vector<int64_t> window_size, const at::Tensor cu_seqlens, const at::Tensor QKV,
     const transformer_engine::DType qkv_type, const c10::optional<at::Tensor> cu_seqlens_padded,
-    const c10::optional<at::Tensor> descale_QKV, const c10::optional<at::Tensor> descale_S,
-    const c10::optional<at::Tensor> scale_S, const c10::optional<at::Tensor> scale_O,
-    c10::optional<at::Tensor> amax_S, c10::optional<at::Tensor> amax_O,
+    const c10::optional<at::Tensor> descale, const c10::optional<at::Tensor> scale,
+    c10::optional<at::Tensor> amax, const int offset_QKV, const int offset_S, const int offset_O,
```

Comment:
I don't think we should assume that all of the scaling factors are packed together. For example, if all the inputs are `Float8Tensor`s, then each one will have a separate `scale_inv` tensor.
```suggestion
    const c10::optional<at::Tensor> descale_QKV, int descale_QKV_offset,
    const c10::optional<at::Tensor> descale_S, int descale_S_offset,
    const c10::optional<at::Tensor> scale_S, int scale_S_offset,
    const c10::optional<at::Tensor> scale_O, int scale_O_offset,
    c10::optional<at::Tensor> amax_S, int amax_S_offset,
    c10::optional<at::Tensor> amax_O, int amax_O_offset,
```